Jenkinsfile

```diff
@@ -433,6 +437,13 @@ lock(resource: "build-${params.STREAM}") {
                     --fedmsg-conf /etc/fedora-messaging-cfg/fedmsg.toml
                 """)
             }
+            stage("OSTree Import") {
```

Comment:
Maybe `Import OSTree: Compose Repo` here and `Import OSTree: Prod Repo` in the other one?

Reply:
should be in the next upload.